packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.3
+
+* Publishes an instance of the plugin to the registrar on macOS, as on iOS.
```

Comment:
im curious who is currently using this instance on iOS

Reply:
Probably nobody (although in practice it could be anyone, in code we don't control). I expect the reason the publish call exists is because of the (deeply unfortunate, IMO, and I want to revisit this API at some point) fact that the only way to get [this engine shutdown call](https://github.com/flutter/packages/blob/719dc9ba6719f98c5446c4bf4d54a4545b2e049b/packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m#L693) is to publish an instance of yourself.

(This PR isn't because we need the publish functionality on macOS, but because I want to minimize forking.)